support-frontend/conf/routes

```diff
@@ -89,6 +89,10 @@ POST /identity/signin-url                                          controllers.I
 GET  /subscribe                                                    controllers.SubscriptionsController.geoRedirect()
 GET  /$country<(uk|us|au|eu|int|nz|ca)>/subscribe                  controllers.SubscriptionsController.landing(country: String)
 
+# ----- Checkout ----- #
+GET  /$country<(uk|us|au|eu|int|nz|ca)>/checkout                   controllers.Application.checkout(country: String, product: Option[String])
```

Comment:
The idea here would be to have `/{geo}/checkout` and potentially `/{geo}` for the landing pages.